### DISCUSSION TITLE
Creating inner shadow like css

### DISCUSSION THREAD

**Discussion by humbleCoder1990** [pixijs/pixijs#7768] [Q&A]
![image](https://user-images.githubusercontent.com/87828192/131859543-aa5c57e3-771a-41d8-997b-f0503da30b1a.png)

How to make inner shadow like CSS's inset box-shadow?

**Comment by ivanpopelyshev** [comment]
Make a filter , use blurFilter as inner filter. It wasn't done in pixi, its possible, im sure many people can do it - but no one actually shared this thing. Assign a bounty on this and wait for a hero :)
